BUILDING.md

```diff
@@ -26,7 +26,7 @@ development environment setup. Please follow [How to Write Go Code](https://gola
 for proper setup. If done correctly, you should have a GOROOT and GOPATH set in the
 environment.
 
-If a Go development environment is setup, one can use `go get` to install the
+If a Go development environment is setup, one can use `go install` to install the
```

Comment:
The instructions themselves below haven't been updated, so I suggest reverting the changes in this file

But we should update the one in `main` to be updated for `go modules` (it still outlines using `GOPATH` etc).

Reply:
reverted.